JASP-Engine/JASP/R/reliabilityAnalysisBayesian.R

```diff
@@ -644,7 +661,7 @@ reliabilityBayesian <- function(jaspResults, dataset, options) {
 	  g <- g + ggplot2::geom_line(data = datPrior, mapping = ggplot2::aes(x = x, y = y),
 	                              linetype = "dashed", size = .85) +
 	           ggplot2::scale_x_continuous(name = nms, breaks = xBreaks, expand = xExpand,
-	                                       limits = c(min(xBreaks), max(xBreaks)))
+	                                       limits = c(range(xBreaks)))
```

Comment:
```suggestion
	                                       limits = range(xBreaks))
```
`c()` doesn't do anything right?